src/UglyToad.PdfPig/Content/InlineImage.cs

```diff
@@ -50,7 +49,7 @@ public class InlineImage : IPdfImage
         public bool Interpolate { get; }
 
         /// <inheritdoc />
-        public IReadOnlyList<byte> RawBytes { get; }
+        public ReadOnlyMemory<byte> RawMemory { get; }
```

Comment:
Use `RawBytes` - see other comment. Unless you intend to expose `public ReadOnlySpan<byte> RawBytes => RawMemory.Span`, if yes - then let's put both

Reply:
Doubled up RawBytes, which should minimize any breaking changes after a compilation.